src/main/resources/sun_checks.xml

```diff
@@ -87,6 +87,7 @@
 
         <!-- Checks for Javadoc comments.                     -->
         <!-- See http://checkstyle.sourceforge.net/config_javadoc.html -->
+        <module name="InvalidJavadocPosition"/>
         <module name="JavadocMethod"/>
         <module name="JavadocType"/>
```

Comment:
Should we name Check "JavadocPosition" ?

Reply:
@pbludov , please share your ideas on this ?
https://checkstyle.org/config_naming.html , we can put "Invalid" to almost all Checks, but we do not do this.
It would be better for us to make some concept of module naming.
We have now that much [Checks with prefixes](https://checkstyle.org/checks.html): "Avoid" , "Illegal", "Missing", "No", "Simplify".

In my mind, javadoc position is very certain, it could not several acceptable positions and some position are invalid. So we just check position on certain place, so "JavadocPosition".

Reply:
to not block this awesome PR that much, lets vote one more time on name.

Reply:
> @pbludov , please share your ideas on this ?

It is a good point. We don't have checks with the prefix `Invalid`. But simple `JavadocPosition` may be not clear enough. By the way, this check is not about *positions*. It is about *relations*. Each javadoc comment should be bound to some code. If there is no connection, it is a violation. That's what this check does. We need to show it in the name of the check somehow.

Reply:
as `JavadocPosition` is not clear also, and you have no better name, lets keep `InvalidJavadocPosition`.
We can not keep PR that long.